doc/getting_started.md

```diff
@@ -1,6 +1,22 @@
 # Getting Started
 
-Although the most straightforward way to use Torque is through either the CartoDB Editor, or by passing the layer's viz.json to [CartoDB.js](http://docs.cartodb.com/cartodb-platform/cartodb-js/getting-started/), many use cases work best with the standalone [Torque.js](https://github.com/CartoDB/torque/tree/master/dist). Assuming you have a public dataset with a `date` column, it is really simple to create an animated map with the library. First, you need to have a Leaflet map prepared in an HTML page:
+Torque.js is a JavaScript library that enables you to animate time series data. Torque.js uses tile cubes, which are JSON representations of multidimensional data with geospatial coordinates. [Tile cube specifications](http://wiki.osgeo.org/wiki/Tile_Map_Service_Specification
```

Comment:
As discussed [here](https://github.com/CartoDB/torque-tiles/issues/8), we're phasing out all terms other than Torque tiles. And the link to the spec is wrong: 

https://github.com/CartoDB/torque-tiles/blob/master/2.0/spec.md